node_groups.tf

```diff
@@ -296,6 +296,7 @@ module "eks_managed_node_group" {
   enable_monitoring                  = try(each.value.enable_monitoring, var.eks_managed_node_group_defaults.enable_monitoring, true)
   network_interfaces                 = try(each.value.network_interfaces, var.eks_managed_node_group_defaults.network_interfaces, [])
   placement                          = try(each.value.placement, var.eks_managed_node_group_defaults.placement, null)
+  launch_template_tags               = try(each.value.launch_template_tags, var.eks_managed_node_group_defaults.launch_template_tags, null)
```

Comment:
woops, one more - the very last default should be `{}` like the other tags are set here, and we can move this up to line 276

Reply:
done